doc/DOCUMENTATION.org

```diff
@@ -3389,54 +3389,77 @@ Examples:
-| ~SPC k L~   | go to next sexp                                                     |
-| ~SPC k p~   | paste after                                                         |
-| ~SPC k P~   | paste before                                                        |
-| ~SPC k r~   | raise expression (replace parent expression by current one)         |
-| ~SPC k s~   | forward slurp expression                                            |
-| ~SPC k S~   | backward slurp expression                                           |
-| ~SPC k t~   | transpose expression                                                |
-| ~SPC k u~   | undo                                                                |
-| ~SPC k U~   | got to parent sexp backward                                         |
-| ~SPC k C-r~ | redo                                                                |
-| ~SPC k v~   | switch to =visual state=                                            |
-| ~SPC k V~   | switch to =visual line state=                                       |
-| ~SPC k C-v~ | switch to =visual block state=                                      |
-| ~SPC k w~   | wrap expression with parenthesis                                    |
-| ~SPC k W~   | unwrap expression                                                   |
-| ~SPC k y~   | copy expression                                                     |
+
+****** Adding parts
+
+| Key binding | Description                                          |
+|-------------+------------------------------------------------------|
+| ~SPC k (~   | insert expression before (same level as current one) |
+| ~SPC k )~   | insert expression after (same level as current one)  |
+| ~SPC k w~   | wrap expression with parenthesis                     |
+
+****** Deleting parts
+
+| Key binding | Description                                                    |
+|-------------+----------------------------------------------------------------|
+| ~SPC k Dx~  | backward delete expression                                     |
+| ~SPC k Dw~  | backward delete word                                           |
+| ~SPC k Ds~  | backward delete symbol                                         |
+| ~SPC k E~   | unwrap current expression and kill all symbols before point    |
+| ~SPC k e~   | unwrap current expression and kill all symbols after point     |
+| ~SPC k ds~  | delete symbol                                                  |
+| ~SPC k dw~  | delete word                                                    |
+| ~SPC k dx~  | delete expression                                              |
+| ~SPC k ` k~ | hybrid version of kill sexp (can be used in non lisp dialects) |
+| ~SPC k r~   | raise expression (replace parent expression by current one)    |
+| ~SPC k W~   | unwrap expression                                              |
+
+****** Other: entering insert, visual state, undo/redo, copy and paste
```

Comment:
This list of bindings are very basic and should be put before the other more complicated ones.

It can be named as `General`

Reply:
I think the navigation commands are the most basic. The collection of commands here are quite special purpose and (except for `i` and `I`) used quite a bit less often than the navigation commands, I believe.

Reply:
Nah. Every bindings in this list exactly the same bindings in evil.
While the bindings in navigation, and others, are generally specialised for LISP sexp.

Reply:
That's a good point. I've pushed a change to account for this.